locale/Makefile

```diff
@@ -4,7 +4,7 @@
 # Run make help to list available targets
 #
 DOMAIN = hammer-cli
-VERSION = $(shell bundle exec ruby -e 'require "rubygems"; spec = Gem::Specification::load("../hammer_cli.gemspec"); puts spec.version')
-MAIN_MAKEFILE = $(shell bundle exec ruby -e 'require "hammer_cli"; puts HammerCLI::I18n.main_makefile')
+VERSION = $(shell ruby -rrubygems -e 'puts Gem::Specification::load(Dir.glob("../*.gemspec").first).version')
+MAIN_MAKEFILE = $(shell bundle exec ruby -rhammer_cli -e 'puts HammerCLI::I18n.main_makefile')
```

Comment:
I guess I know the difference: in the old version `require "hammer_cli"` is executed in context of `bundle exec` loading local gem, but in the new version `-rhammer_cli` is executed in context of `ruby` ignoring `bundle exec` and since there is no installed `hammer_cli` gem it fails.

It does work for `VERSION` since `rubygems` is available globally.